wflambda.go

```diff
@@ -0,0 +1,283 @@
+package wflambda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambdacontext"
+	"github.com/rcrowley/go-metrics"
+	"github.com/wavefronthq/go-metrics-wavefront"
+	"log"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+)
+
+type lambdaHandlerFunction func(context.Context, json.RawMessage) (interface{}, error)
+
+var (
+	server                  string
+	authToken               string
+	isReportStandardMetrics bool
+	csCounter               metrics.Counter
+	csEventCounter          metrics.Counter
+	invocationsCounter      metrics.Counter
+	invocationEventCounter  metrics.Counter
+	eCounter                metrics.Counter
```

Comment:
Assuming these are for errors, let's just name them `errCounter` and `errEventCounter` to make it clear.